packages/formik/src/FieldArray.tsx

```diff
@@ -157,25 +157,38 @@ class FieldArrayInner<Values = {}> extends React.Component<
       let updateTouched =
         typeof alterTouched === 'function' ? alterTouched : fn;
 
+
+      let values = setIn(
```

Comment:
I have to bring values up, to be executed before  `updateErrors` and `updateTouched`. Otherwise, it would cause an error with `unshift`.

Reply:
Add this in a comment plz above these lines